panther_hardware_interfaces/README.md

```diff
@@ -52,13 +31,9 @@ That said apart from the usual interface provided by the ros2_control, this plug
 - `hardware/led_control_enable` [*std_srvs/SetBool*]: Enables or disables SBC (Single Board Computer) control over the LEDs.
 - `hardware/motor_power_enable` [*std_srvs/SetBool*]: Enables or disables motor power.
 
-[//]: # (ROS_API_NODE_SERVICE_SERVERS_END)
-
 #### Parameters
 
-[//]: # (ROS_API_NODE_PARAMETERS_START)
-
-Parameters that are required, are defined when including interface in URDF (you can check out [panther_macro.urdf.xacro](https://github.com/husarion/panther_ros/panther_description/urdf/panther_macro.urdf.xacro)).
+Parameters that are required, are defined when including interface in URDF (you can check out [panther_macro.urdf.xacro](../panther_description/urdf/panther_macro.urdf.xacro)).
```

Comment:
Yes! ;)
I added this functionality so that we don't have to remember too many things when writing documentation. (the exception are the `<` `>` characters, which are reserved for html.